### PR TITLE
Add GUNC for additional QC  

### LINKED ISSUE #387: Add GUNC as additional MAG quality check
### Description of feature

As requested by @alexhbnr, now we have checkM in the pipeline

Should also merge with CHECKM output using `gunc merge_checkm`

Docs in https://grp-bork.embl-community.io/gunc/index.html

### PR DESCRIPTION
**Pull request by jfy133**
Closes #387 

Also updates the diagram to include COCONCT that was missing from the previous PR

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -455,7 +455,9 @@ Besides the reference files or output files created by BUSCO, the following summ
 
 </details>
 
-### QC for metagenome assembled genomes with CheckM
+### QC for metagenome assembled genomes
```
Comment (d4straub) [inline docs/output.md:458]: Wouldnt that headline be better at line 419 (`### QC for metagenome assembled genomes with BUSCO`) followed by `#### BUSCO` ?

**Review by d4straub — APPROVED** [review]
I have no objections, but it might be good to add a note in the CHANGELOG?

**Comment by jfy133** [comment]
Of all things to forget - and agree with the header change!

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#366](https://github.com/nf-core/mag/pull/366) - Added CAT_SUMMARISE process and cat_official_taxonomy parameter (by @prototaxites)
 - [#372](https://github.com/nf-core/mag/pull/372) - Allow CAT_DB to take an extracted database as well as a tar.gz file.
 - [#380](https://github.com/nf-core/mag/pull/380) - Added support for saving processed reads (clipped, host removed etc.) to results directory (by @jfy133)
+- [#394](https://github.com/nf-core/mag/pull/380) - Added GUNC for additional chimeric bin/contamination QC (added by @jfy13)
```
Comment (d4straub) [inline CHANGELOG.md:16]:
```suggestion
- [#394](https://github.com/nf-core/mag/pull/394) - Added GUNC for additional chimeric bin/contamination QC (added by @jfy13)
```

**Pull request merged**